src/openrct2/peep/PeepData.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "../entity/Peep.h"
+
+#include <vector>
+
+struct SpriteBounds
+{
+    uint8_t sprite_width;           // 0x00
+    uint8_t sprite_height_negative; // 0x01
+    uint8_t sprite_height_positive; // 0x02
+};
+
+struct PeepAnimation
+{
+    const uint32_t base_image;
+    const SpriteBounds bounds;
+    const std::vector<uint8_t> frame_offsets;
+
+    size_t num_frames() const
```

Comment:
numFrames

Reply:
Yeah this was just a quick stop gap solution to replace the old `num_frames` attribute. I'd like to change all property names to match the code style.